.buildkite/pipeline.yaml

```diff
@@ -40,6 +40,10 @@ steps:
     artifact_paths: _out_/profile.json
     <<: *elastic
 
+  - label: ":linux: test-sorbet-rubyfmt.sh"
+    command: .buildkite/test-sorbet-rubyfmt.sh
+    <<: *elastic
+
```

Comment:
Curious what you think about moving this to be a job that happens down at the bottom of the file (next to the other non-blocking job)?

Right now as written this is still a blocking job.

Reply:
Ah, that was the intention originally. I had thought that always returning `0` would make it so that the job could fail, but that the failure wouldn't hold up subsequent jobs.

Reply:
Ah yeah that would work, but it would still potentially spend an instance running these tests before you could click the merge button.